model/Core/Vocabularies/RelationshipType.md

```diff
@@ -12,73 +12,79 @@ Provides information about the relationship between two Elements.
-- dataFile: Every `to` Element is a data file related to the the `from` Element
-- declaredLicense: The `to` license information identifies the license information actually found in the `from` software Artifact, for example as detected by use of automated tooling.
-- dependencyManifest: Every `to` Element is manifest file containing dependency information related to the `from` Element
-- dependsOn: Every `to` Element is a dependecy of the `from` Element
-- descendant: This relationship may be used to describe child builds of a Build instance.
-- describes: Every `to` Element is  described by the `from` Element.  This can be used to denote the root(s) of a tree of elements contained in an SBOM.
-- devDependency: Every `to` Element is a development dependency for the `from` Element
-- devTool: Every `to` Element is a development tool for the `from` Element
-- distributionArtifact: Every `to` Element is an artifact intended for distribution of the `from` Element (e.g. an RPM or archive file)
-- documentation: Every `to` Element is documentation for the `from` Element
-- doesNotAffect: (Security/VEX) Specifies a vulnerability has no impact on one or more elements
-- dynamicLink: Every `to` Element is dynamically linked to the `from` Element
-- example: Every `to` Element is an example for the `from` Element
-- evidenceFor: (Dataset) Every `to` Element is can be considered as evidence for the `from` Element
-- expandedFromArchive: Every `to` Element is an artifact expanded from the `from` archive file
+- hasConcludedLicense: The `from` software Artifact is concluded by the SPDX data creator to be governed by each `to` license
+- contains: The `from` Element contains each `to` Element
+- configOf: The `from` Element is a configuration applied to each `to` Build during a LifecycleScopeType period
+- copiedTo: The `from` Element has been copied to each `to` Element
+- hasDataFile: The `from` Element treats each `to` Element as a data file
+- hasDeclaredLicense: The `from` software Artifact was discovered to actually contain each `to` license, for example as detected by use of automated tooling.
+- hasDependencyManifest: The `from` Element has manifest files that contain dependency information in each `to` Element
+- dependsOn: The `from` Element depends on each `to` Element
+- descendantOf: The `from` Element is a descendant of each `to` Element
+- describes: The `from` Element describes each `to` Element. This can be used to denote the root(s) of a tree of elements contained in an SBOM.
+- hasDevDependency: The `from` Element has a development dependency on each `to` Element
+- hasDevTool: The `from` Element uses each `to` Element as a development tool
+- hasDistributionArtifact: The `from` Element is distributed as an artifact in each Element `to`, (e.g. an RPM or archive file)
+- hasDocumentation: The `from` Element is documented by each `to` Element
+- doesNotAffect: The `from` (Security/VEX) vulnerability has no impact on each `to` Element
```

Comment:
Please use: "(Security/VEX) The `from` Vulnerability element does not affect each `to` Element (i.e. product)"